docs/current/cli/465058-install.md

```diff
@@ -0,0 +1,129 @@
+---
+slug: /cli/465058/install
+---
+
+# Installation
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+import CodeBlock from "@theme/CodeBlock";
+import styles from "@site/src/css/install.module.scss";
+
+:::tip
+The `dagger` CLI is available for installation on macOS, Linux, and Windows to run locally or in a CI environment.
+:::
+
+Install the `dagger` CLI following the steps below.
+
+<BrowserOnly>
+{() =>
+<Tabs defaultValue={
+ window.navigator.userAgent.indexOf('Linux') != -1 ? 'linux':
+ window.navigator.userAgent.indexOf('Win') != -1 ? 'windows':
+ 'macos'}
+groupId="os"
+values={[
+{label: 'macOS', value: 'macos'}, {label: 'Linux', value: 'linux'}, {label: 'Windows', value: 'windows'},
+]}>
+
+<TabItem value="macos">
+
+We assume that you have [Homebrew](https://brew.sh/) installed.
+If you do, you can install `dagger` with a single command:
+
+```shell
+brew install dagger
+```
+
+This installs `dagger` in:
+
+```shell
+type dagger
+# macOS ARM:
+dagger is /opt/homebrew/bin/dagger
+# macOS Intel:
+dagger is /usr/local/bin/dagger
+```
+
+If you do not have Homebrew installed, or you want to install a specific version of `dagger`, you can run:
+
+```shell
+curl -L https://dl.dagger.io/dagger/install.sh | VERSION=0.2.232 sh
+
+./bin/dagger version
+dagger 0.2.232 (GIT_SHA) darwin/arm64
```

Comment:
This version will be different, we should update it once we release the new CLI.